sdcm/cluster.py

```diff
@@ -3021,6 +3027,7 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster', node_prefix='nod
             self.add_nodes(n_nodes, enable_auto_bootstrap=self.auto_bootstrap)
         else:
             raise ValueError('Unsupported type: {}'.format(type(n_nodes)))
+        self.run_node_benchmarks()
```

Comment:
in doc file, you mentioned that this will be run only during cluster initialization, but call run benchmark in __init__, So the benchmark will be executed for each node which is added during nemesis, won't be?

Reply:
It shouldn't - it's the `__init__` method of the `Base Cluster` class, so my understanding is it will only be called when a cluster is being created. If a nemesis class creates an additional cluster, then yes, it would call the benchmark methods on that additional cluster.